app/models/invenio_rdm_record_converter.rb

```diff
@@ -547,4 +547,29 @@ def format_prism_community_collection_string(mapping_entry)
       community_id
     end
   end
+
+  def original_identifiers(identifiers)
+    identifiers.map do |identifier|
+      if identifier.include?("PMID")
+        id = identifier.gsub(/\(PMID\)/, "").strip
+        scheme = "pmid"
+      elsif identifier.include?("DOI")
+        id = identifier.gsub(/\("DOI"\)/, "").strip
+        scheme = "doi"
+      elsif identifier.include?("ISBN")
+        id = identifier.gsub(/\(ISBN}.*\)/, "").strip
+        scheme = "isbn"
+      elsif identifier.include?("PNB")
+        id = identifier
+        scheme = "other"
+      else
+        next
+      end
+
+      {
+        "identifier": id,
+        "scheme": scheme
+      }
+    end
```

Comment:
Put the `compact` call from L186 in the method itself to remove dangling operation.

Reply:
good call